source/shared/xplat.h

```diff
@@ -162,59 +85,19 @@ using std::max;
 //
 // These types are used in this file primarily to define common Windows types (DWORD, LONG, etc.)
 // Cross-platform code should use either the Windows types or appropriate types from <stdint.h>.
-#if defined(_MSC_VER) // WwoWH
-typedef long windowsLong_t;
-typedef unsigned long windowsULong_t;
-typedef __int64 windowsLongLong_t;
-typedef unsigned __int64 windowsULongLong_t;
-#else // *nix (!WwoWH)
 #include <stdint.h> // Use standard bit-specific types (signed/unsigned integrated)
 typedef int32_t windowsLong_t;
 typedef uint32_t windowsULong_t;
 typedef int64_t windowsLongLong_t;
 typedef uint64_t windowsULongLong_t;
-#endif
-typedef windowsLong_t LONG, *PLONG, *LPLONG;
+typedef windowsLong_t LONG;
```

Comment:
They're not used anywhere in the driver code, so right now those typedefs are not needed.